styles.css

```diff
@@ -112,6 +147,11 @@ form {
     align-items: center;
 
     transition: transform 500ms ease-in-out;
+
+    @media (max-width:1200px) {
+        font-size: 14px;
+    }
+
 }
 
 .randomize-btn:hover {
```

Comment:
This `:hover` effect is fun, and I do not mind it in this project. But I do not recommend to use it in the future, at least not for buttons: it makes them harder to click.